docs/backend-requests/making/custom-session-token.mdx

```diff
@@ -15,35 +15,18 @@ This guide will show you how to customize a session token to include additional
   The entire session token has a max size of 4kb. Exceeding this size can have adverse effects, including a possible infinite redirect loop for users who exceed this size on Next applications.
 </Callout>
 
-## How to customize your session token
+{/* TODO: Update the H3 to an H2 when the Steps component can accept headings other than H3. */}
 
 <Steps>
 
-### Go to **Sessions** in the Clerk Dashboard
+### Add custom claims to your session token
 
-In the Clerk Dashboard, navigate to the **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)** page.
+1. Navigate to your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=sessions).
```

Comment:
We should remove the periods at the ends of these list items, right?

Reply:
It's not a fragment sentence, it's a full sentence